__tests__/api.test.js

```diff
@@ -89,7 +89,62 @@ describe('GET /api/articles/:article_id', () => {
         });
     })
 })
-
+describe('POST /api/articles/:article_id/comments', () => {
+    test("returns the new comment that has been created along with 201 status", () => {
+        return request(app)
+        .post("/api/articles/2/comments")
+        .send({
+            username: "butter_bridge",
+            body: "well well well"
+        })
+        .expect(201)
+        .then(({body}) => {
+            const comment = body.comment[0]; 
+            expect(comment).toMatchObject({ 
+                body: "well well well",
+                votes: expect.any(Number),
+                article_id: 2,
+                author: "butter_bridge",
+                comment_id:  expect.any(Number)
+            });
+        });
+    })
```

Comment:
Please add another test where the extra properties in the post object are ignored. For example when posting with this:
```
{
            username: "butter_bridge",
            body: "well well well",
            extraProp: "extraValue"
        }
```

Reply:
done